doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,102 @@
+============================================
+Cross Validation
+============================================
+
+Cross validation is a type of model validation technique to estimate an algorithm's performance on unseen data.
+
+It essentially involves training the model on a subset of the data, and testing on the
+remaining data.
+
+The process is repeated for several complementary partitions and
+combines all the validation results generated (For instance, an average).
+
+One type of cross validation is known as :math:`k-` fold cross validation.
+
+Suppose we have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{N}`
+for a positive integer :math:`n` , then :math:`k-` fold cross validation partitions
+the data into :math:`P_1, P_2, ..., P_n` each with equal size (Where :math:`n=\frac{N}{k}`).
+
+Next, the model would be trained on :math:`P_1, P_2, ...,P_{n-1}` and is tested on :math:`P_n` to get performance measure
+:math:`a_1` .
+
```

Comment:
remove whitespace, make this one paragraph

Reply:
Dont use tenses like "would be trained", just say "is trained" (also in the other places)